core/iwasm/aot/arch/aot_reloc_aarch64.c

```diff
@@ -53,7 +53,11 @@ get_target_symbol_map(uint32 *sym_num)
     return target_sym_map;
 }
 
+#if !__MACH__
```

Comment:
How about `!defined(__MACH__)`? `!__MACH__` may cause compilation error on some platforms since `__MACH__` is not defined.